ui/components/MesheryAdapterPlayComponent.js

```diff
@@ -389,19 +390,39 @@ class MesheryAdapterPlayComponent extends React.Component {
 
 
   generateSMIResult() {
+    const self = this;
+    
     const {
       customDialogSMI,
-    } = this.state;
-    const columns = ["Test", "SMI Version", "Service Mesh", "Service Mesh Version", "SMI Specification", "Capability", "Test Status"];
-
-    const data = [
-      ["TA-01", "v1alpha3", "Linkerd", "edge-20.7.5", "Traffic Access", "Full", "Passed"],
-      ["TA-02", "v1alpha3", "Linkerd", "edge-20.7.5", "Traffic Access", "Full", "Failed"],
-      ["TM-01", "v1alpha3", "Linkerd", "edge-20.7.5", "Traffic Metrics", "Half", "Passed"],
-      ["TM-02", "v1alpha3", "Linkerd", "edge-20.7.5", "Traffic Metrics", "None", "Passed"],
-      ["TM-03", "v1alpha3", "Maesh", "v1.3.2", "Traffic Metrics", "None", "Failed"],
-      ["TM-04", "v1alpha3", "Maesh", "v1.3.2", "Traffic Metrics", "Full", "Passed"],
-    ];
+    } = self.state;
+    
+    const columns = ["Id", "Mesh name", "Mesh version", "Cases passed", "Conformance capability", "Status", "More details"];
```

Comment:
Capitalize the column names, please. If we're aggregating all the tests into a single line, here's a good format for that line:

["SMI Specification", "SMI Version", "Service Mesh", "Service Mesh Version", "% Passed", "Details"];

Consolidating into one line has the effect of potentially needed a table-in-a-table user interface.

Only aggregating tests at an SMI specification category level means that a single table user interface could be presented. It does have the effect of showing the user four lines per conformance test run. This second design is what we originally intended to display to the user. The SMI test results table mockup previously created is here - https://docs.google.com/spreadsheets/d/1yTGPgqbG_QrVd3ENEdCmqwQwFJMNAXCgcil6crcxvhM/edit#gid=0

// @dhruv0000

Reply:
Of the latest screenshots that I've seen, I think we're good to go here.